api/dbadapters/bigquery.ts

```diff
@@ -1,25 +1,29 @@
-import * as protos from "@dataform/protos";
+import { dataform } from "@dataform/protos";
 import * as PromisePool from "promise-pool-executor";
-import { DbAdapter, OnCancel } from "./index";
+import { DbAdapter, OnCancel } from "df/api/dbadapters/index";
+import * as util from "df/api/utils";
 
 const BigQuery = require("@google-cloud/bigquery");
 
 export class BigQueryDbAdapter implements DbAdapter {
-  private profile: protos.IProfile;
+  private bigquery: dataform.IBigQuery;
```

Comment:
should this be called credentials or bigquery credentials? it's a bit confusing as we have the BigQuery class that we require above too.

Reply:
Done